Dockerfiles/lambda/Makefile

```diff
@@ -0,0 +1,25 @@
+.phony: all
+
+all: docker-build docker-push
+
+IMG_NAME = PROJECT-dashboard-api-base
```

Comment:
Should `PROJECT` be `$PROJECT` here?

Reply:
Ah I see below, instructions are to update this file directly. Is it possible for `PROJECT` to be set as an environment variable before running?

Reply:
Since this is a template for a project, I think it's better to change it permanently in the code, rather than rely on an env var being set each time you run it.